<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>qG3PointDialog</class>
 <widget class="QDialog" name="qG3PointDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>357</width>
    <height>608</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_5">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Segmentation</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QGroupBox" name="groupBox_3">
         <property name="title">
          <string>Initial segmentation</string>
         </property>
         <layout class="QFormLayout" name="formLayout">
          <item row="1" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>kNN</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QSpinBox" name="spinBoxkNN">
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>100000</number>
            </property>
            <property name="value">
             <number>20</number>
            </property>
           </widget>
          </item>
          <item row="2" column="0" colspan="2">
           <widget class="QPushButton" name="pushButtonSegment">
            <property name="text">
             <string>Segment</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBoxClusterAndOrClean">
         <property name="title">
          <string>Cluster and / or clean</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QGroupBox" name="groupBox_4">
            <property name="title">
             <string>Cluster</string>
            </property>
            <layout class="QFormLayout" name="formLayout_5">
             <item row="1" column="0">
              <widget class="QLabel" name="label_4">
               <property name="text">
                <string>Radius factor</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QDoubleSpinBox" name="doubleSpinBoxRadiusFactor">
               <property name="value">
                <double>0.600000000000000</double>
               </property>
              </widget>
             </item>
             <item row="2" column="0">
              <widget class="QLabel" name="label_2">
               <property name="text">
                <string>Max angle 1</string>
               </property>
              </widget>
             </item>
             <item row="2" column="1">
              <widget class="QDoubleSpinBox" name="doubleSpinBoxMaxAngle1">
               <property name="frame">
                <bool>true</bool>
               </property>
               <property name="buttonSymbols">
                <enum>QAbstractSpinBox::UpDownArrows</enum>
               </property>
               <property name="maximum">
                <double>360.000000000000000</double>
               </property>
               <property name="value">
                <double>60.000000000000000</double>
               </property>
              </widget>
             </item>
             <item row="0" column="0">
              <widget class="QCheckBox" name="checkBoxClustering">
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="groupBox_5">
            <property name="title">
             <string>Clean</string>
            </property>
            <layout class="QFormLayout" name="formLayout_6">
             <item row="1" column="0">
              <widget class="QLabel" name="label_5">
               <property name="text">
                <string>Max angle 2</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QDoubleSpinBox" name="doubleSpinBoxMaxAngle2">
               <property name="value">
                <double>10.000000000000000</double>
               </property>
              </widget>
             </item>
             <item row="2" column="0">
              <widget class="QLabel" name="label_6">
               <property name="text">
                <string>Min points per grain</string>
               </property>
              </widget>
             </item>
             <item row="2" column="1">
              <widget class="QSpinBox" name="spinBoxNMin">
               <property name="maximum">
                <number>1000000000</number>
               </property>
               <property name="value">
                <number>50</number>
               </property>
              </widget>
             </item>
             <item row="3" column="0">
              <widget class="QLabel" name="label_7">
               <property name="text">
                <string>Min flatness</string>
               </property>
              </widget>
             </item>
             <item row="3" column="1">
              <widget class="QDoubleSpinBox" name="doubleSpinBoxMinFlatness">
               <property name="singleStep">
                <double>0.100000000000000</double>
               </property>
               <property name="value">
                <double>0.100000000000000</double>
               </property>
              </widget>
             </item>
             <item row="0" column="0">
              <widget class="QCheckBox" name="checkBoxCleaning">
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButtonClusterAndOrClean">
            <property name="text">
             <string>Cluster and / or clean</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButtonGetBorders">
            <property name="text">
             <string>Get borders</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>Cloud</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelCloud">
         <property name="text">
          <string>-</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Fitting</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>Fit ellipsoids</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QPushButton" name="pushButtonFit">
            <property name="text">
             <string>Fit</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="radioButtonDirect">
            <property name="text">
             <string>Direct</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Show</string>
         </property>
         <layout class="QFormLayout" name="formLayout_2">
          <item row="0" column="0">
           <widget class="QRadioButton" name="radioButtonAll">
            <property name="text">
             <string>All</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QRadioButton" name="radioButtonOnlyOne">
            <property name="text">
             <string>Only one</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QSpinBox" name="spinBoxOnlyOne">
            <property name="maximum">
             <number>1000</number>
            </property>
            <property name="value">
             <number>0</number>
            </property>
           </widget>
          </item>
          <item row="2" column="0" colspan="2">
           <widget class="QCheckBox" name="checkBoxWireframes">
            <property name="text">
             <string>Wireframes</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QCheckBox" name="checkBoxSurfaces">
            <property name="text">
             <string>Surfaces</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QCheckBox" name="checkBoxPoints">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Points</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QSpinBox" name="spinBoxGLPointSize">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="maximum">
             <number>16</number>
            </property>
            <property name="value">
             <number>3</number>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonExportResults">
         <property name="text">
          <string>Export results</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>Dev</string>
      </attribute>
      <layout class="QFormLayout" name="formLayout_4">
       <item row="0" column="0">
        <widget class="QRadioButton" name="radioButtonG3Point">
         <property name="text">
          <string>G3Point</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QRadioButton" name="radioButtonSteepestSlope">
         <property name="text">
          <string>steepest slope</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>24</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
